react/src/components/DeleteMapModal/DeleteMapModal.tsx

```diff
@@ -15,12 +16,35 @@ const DeleteMapModal = ({
   close: parentToggle,
   project,
 }: DeleteMapModalProps) => {
+  const navigate = useNavigate();
+  const location = useLocation();
   const {
     mutate: deleteProject,
     isPending: isDeletingProject,
     isError,
     isSuccess,
   } = useDeleteProject();
+
```

Comment:
i think we can do the work currently in the `useEffect` on the mutate/deleteProjects' `onSuccess`.  would make it clearer here and in the .  This would probably . so can discuss on slack if we just do this as a follow on.

Reply:
Done! [WG-430](https://tacc-main.atlassian.net/browse/WG-430)